test/singleRouteSSE.spec.js

```diff
@@ -185,12 +185,61 @@ describe('EventSource implementation for resource changes', function () {
 
                     expect(_.omit(data, 'id')).to.deep.equal(payloads[counter].books[0]);
                     counter ++;
-                    if (counter === 1) {
+                    if (counter === 2) {
```

Comment:
With `1` this test leve `it` too early and actully didn't test what it should test